simulation.hpp

```diff
@@ -342,7 +355,8 @@ class Simulation
                   std::vector<std::tuple<int, int>>& outside_dispersers,
                   bool weather,
                   const FloatRaster& weather_coefficient,
-                  DispersalKernel& dispersal_kernel)
+                  DispersalKernel& dispersal_kernel,
+                  double establishment_probability = 1)
```

Comment:
I think the default should be 0.5 as it would allow for some establishment, otherwise, everything >0 will establish. This is good for general testing of the functionality via tests but not ideal for testing the effect of the random effects on uncertainty outcomes (this is our most likely use case).

Reply:
Makes sense. Done and documented.